src/desktop/components/main_layout/stylesheets/body.styl

```diff
@@ -74,6 +74,8 @@ html[data-useragent*="iPad"]
     display none
   #main-layout-header-home-logo
     border-right 1px solid gray-lighter-color
+  #main-layout-container
+    margin-top 42px !important
```

Comment:
This seems like it could affect other apps (like consignments)-- just confirming this looks correct in all cases?

Reply:
It does affect other apps - e.g. some  `auction_support` and `order` templates. I wasn't sure how to access some of the routes, though it does look correct for consignments and `order2`. I can scope it to just those pages for now, to play it safe.

Reply:
That would probably be best until we can confirm (they may also have different requirements).